.prow.yaml

```
@@ -708,21 +708,21 @@ presubmits:
             - e2e-test
           env:
             - name: PROVIDER
-              value: "packet"
+              value: "equinixmetal"
             - name: TEST_CLUSTER_TARGET_VERSION
               value: "1.20.10"
             - name: KUBEONE_TEST_RUN
               value: "TestClusterConformance"
           resources:
             requests:
               cpu: 1
-  - name: pull-kubeone-e2e-packet-conformance-1.21
+  - name: pull-kubeone-ewe-equinix-metal-conformance-1.21
```

Typo -- there's such a typo in a few other places in this file.

```suggestion
  - name: pull-kubeone-e2e-equinix-metal-conformance-1.21
```

---

Thanks, I'll take care of this